app/lib/screens/HomeScreens/News.dart

```diff
@@ -28,8 +29,10 @@ class _NewsScreenState extends State<NewsScreen> {
           builder: (BuildContext context) {
             return IconButton(
               icon: Container(
+                height: MediaQuery.of(context).size.height,
+                width: MediaQuery.of(context).size.width,
                 margin: const EdgeInsets.only(bottom: 10, left: 10),
-                child: Image.asset('assets/images/hamburger.png'),
+                child: SvgPicture.asset('assets/images/hamburger.svg'),
```

Comment:
this is not the right image now. We wanted the menu-lined-`e`, this is now a classic hamburger menu. Please revert.